js/vod-stream-embed.js

```diff
@@ -0,0 +1,23 @@
+// Video-on-demand Stream Video
+
+var ipfs_gateway_self = 'https://ipfs.io'; // IPFS gateway of this node
+var ipfs_gateway_origin = 'https://ipfs.io'; // IPFS gateway of origin stream
```

Comment:
Not following what this is for and how it corresponds to line 11. We're concating to one gateway?

Reply:
This allows for swapping out the gateway. This is only relevant for livestream and is defined here for consistency.